docs/deployment/plugins/k8s/index.rst

```diff
@@ -550,6 +348,26 @@ Specify plugin configuration
 
     .. tabs::
 
+      .. group-tab:: Demo (sandbox) cluster
+
+         Enable the Ray plugin on the demo cluster by adding the following block to ~/.flyte/sandbox/config.yaml:
+
+        .. code-block:: yaml
```

Comment:
needs to be indented.

Reply:
Not sure I follow and the local `make` doesn't detect indentation drift. What specific line here are you referring to?

Reply:
shouldn't there be one extra space?
```suggestion
         .. code-block:: yaml
```

Reply:
my bad. there's an extra space at the start in the "Enable the Ray ..." line. `code-block` is correctly indented.